test/generators.js

```diff
@@ -3,12 +3,24 @@ var jsc = require("jsverify");
 // this is just to have web3 available and correctly initialized
 artifacts.require("./LifToken.sol");
 
-const accountGen = jsc.nat(web3.eth.accounts.length - 1);
+const knownAccountGen = jsc.nat(web3.eth.accounts.length - 1);
+const zeroAddressAccountGen = jsc.constant("zero");
```

Comment:
this should be `0x0000000000000000000000000000000000000000` instead of zero, that is and empty address on the EVM.